crn_robot_description:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  wheel_front_left_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_1_
    pid: {p: 10.0, i: 0.0, d: 1.0}

  wheel_front_right_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_2_
    pid: {p: 10.0, i: 0.0, d: 1.0}

  wheel_rear_left_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_3_
    pid: {p: 10.0, i: 0.0, d: 1.0}

  wheel_rear_right_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_4_
    pid: {p: 10.0, i: 0.0, d: 1.0}

  gazebo_ros_control:
    pid_gains:
      wheel_1 : {p: 10.0, i: 0.0, d: 1.0}
      wheel_2 : {p: 10.0, i: 0.0, d: 1.0}
      wheel_3 : {p: 10.0, i: 0.0, d: 1.0}
      wheel_4 : {p: 10.0, i: 0.0, d: 1.0}
